examples/opentelemetry/docker-compose.yaml

```diff
@@ -69,3 +69,6 @@ services:
     command: ["--config=/etc/otel-collector-config.yaml"]
     ports:
     - "${PORT_UI:-55679}:55679"
+    volumes:
+      - "./otel-collector-config.yaml:/etc/otel-collector-config.yaml"
```

Comment:
i think this isnt necessary and is not what we want - probably it wont work

for container configs - at least where they run as non-root - the configs are loaded in the Dockerfile recipe and chmodded/owned - the reason for this is that some users have a strict umask set so loading from the filesystem as a volume doesnt work

Reply:
Ah didn't know about it